src/main/java/com/simibubi/create/foundation/CreateNBTProcessors.java

```diff
@@ -17,12 +21,15 @@
 
 public class CreateNBTProcessors {
 	public static void register() {
-
+		// Remove the first layer of commands while preserving the styles.
+		// Since recursive commands won't be executed, there's no need to handle them.
```

Comment:
bb4e659740f81289a61eabaa6e35892d159620a1 has handled them anyhow, so this comment is now wrong.
That same commit also made the `tryRemovingCommand` method you added redundant.